UKHomeOffice/nmsw-ui
# NMSW-463 Multifile Upload - remove files from pending list

# Ticket

NMSW-463

----

## AC

Given I have selected one or more files for upload
When I click the delete button next to one
Then it is removed from the files to upload list


----

## To test

- add one or more files to supporting documents page
- click delete next to one
- > it should be removed from the list

----

## Notes
